stl/athena/killing11l/doc/task_1/README.md

```diff
@@ -1 +1,63 @@
-## Task1 的学习笔记
\ No newline at end of file
+## Task1 的学习笔记
+# std::List默认构造器std::allocator<T>
```

Comment:
markdown grammer sequence.

#. -> first-level title
## ->  second-level
###....

Reply:
Ok, the modification has been submitted to the team for review.